phpcs.xml

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0"?>
```

Comment:
This file should not be added, instead changes should be made to `phpcs.xml.dist`